CoreScriptsRoot/CoreScripts/Topbar.lua

```diff
@@ -727,6 +714,10 @@ local function CreateSettingsIcon(topBarInstance)
 		UpdateHamburgerIcon()
 	end)
 
+	HamburgerTransparencyCallback = function(transparency)
```

Comment:
Lets actually subclass CreateSettingsIcon to Create3DSettingsIcon that has this callback as a member function on it instead.